df_to_azure/export.py

```diff
@@ -197,7 +197,7 @@ def upload_to_blob(self):
             blob=f"{self.table_name}/{self.table_name}.csv",
         )
 
-        data = self.df.to_csv(index=False, sep="^", quotechar='"', line_terminator="\n")
+        data = self.df.to_csv(index=False, sep="^", quotechar='"', lineterminator="\n")
```

Comment:
side note: Ive created e a new issue  #125 for making this an export to .parquet.